frontend/src/routes/pipelines/experiments.ts

```diff
@@ -64,3 +64,19 @@ export const experimentScheduleDetailsRoute = (
   !experimentId || !recurringRunId
     ? experimentsBaseRoute(namespace)
     : `${experimentSchedulesRoute(namespace, experimentId)}/${recurringRunId}`;
+
+const compareRunsSelect = (runIds: string[]) => `?compare=${runIds.join(',')}`;
```

Comment:
nit: I'm not sure if the name `compareRunsSelect` for this constant describes the value very well.